README.md

```diff
@@ -87,11 +87,11 @@ Note: In this configuration, you must do some additional configuration:
 - If you wish your Plex Media Server to be accessible outside of your home network, you must manually setup port forwarding on your router to forward to the `ADVERTISE_IP` specified above.  By default you can forward port 32400, but if you choose to use a different external port, be sure you configure this in Plex Media Server's `Remote Access` settings.  With this type of docker networking, the Plex Media Server is essentially behind two routers and it cannot automatically setup port forwarding on its own.
 - (Plex Pass only) After the server has been set up, you should configure the `LAN Networks` preference to contain the network of your LAN.  This instructs the Plex Media Server to treat these IP addresses as part of your LAN when applying bandwidth controls.  The syntax is the same as the `ALLOWED_NETWORKS` below.  For example `192.168.1.0/24,172.16.0.0/16` will allow access to the entire `192.168.1.x` range and the `172.16.x.x` range.
 
-### Using `docker-compose` on ARM devices
+### Using `docker compose` on ARM devices
 
-The provided `docker-compose` templates use the `plexinc/pms-docker` image from [Dockerhub](https://hub.docker.com/r/plexinc/pms-docker) which is currently only build for `amd64` and won't work on ARM devices.
+The provided `docker compose` templates use the `plexinc/pms-docker` image from [Dockerhub](https://hub.docker.com/r/plexinc/pms-docker) which is currently only build for `amd64` and won't work on ARM devices.
```

Comment:
this isnt correct anymore, we build for arm as well, we should probably update this at the same time

Reply:
Removed the whole "compose on ARM" section